File: go/sprint1_nonfinals/h.go
package main

import (
	"bufio"
	"fmt"
	"os"
	"strconv"
)

func getSum(firstNumber string, secondNumber string) string {
	// Ваше решение
}

func main() {
	scanner := makeScanner()
	firstNumber := readLine(scanner)
	secondNumber := readLine(scanner)
	sum := getSum(firstNumber, secondNumber)
	fmt.Println(sum)
}

func makeScanner() *bufio.Scanner {
	const maxCapacity = 3 * 1024 * 1024
	buf := make([]byte, maxCapacity)
	scanner := bufio.NewScanner(os.Stdin)
	scanner.Buffer(buf, maxCapacity)
	return scanner
}

func readLine(scanner *bufio.Scanner) string {
	scanner.Scan()
	return scanner.Text()
}

func printArray(arr []int) {
	writer := bufio.NewWriter(os.Stdout)
	for i := 0; i < len(arr); i++ {
		writer.WriteString(strconv.Itoa(arr[i]))
		writer.WriteString(" ")
	}
	writer.Flush()
}

File: python/sprint1_nonfinals/h.py
from typing import Tuple

def get_sum(first_number: str, second_number: str) -> str:
    # Здесь реализация вашего решения
    pass

def read_input() -> Tuple[str, str]:
    first_number = input().strip()
    second_number = input().strip()
    return first_number, second_number

first_number, second_number = read_input()
print(get_sum(first_number, second_number))
